org-code-javabuilder/theater/src/main/java/org/code/theater/Stage.java

```diff
@@ -325,7 +356,11 @@ public void play() {
     } else {
       this.gifWriter.writeToGif(this.image, 0);
       this.gifWriter.close();
-      outputAdapter.sendMessage(ImageEncoder.encodeStreamToMessage(this.outputStream));
+      this.outputAdapter.sendMessage(ImageEncoder.encodeStreamToMessage(this.imageOutputStream));
+
+      this.audioWriter.writeToAudioStreamAndClose();
```

Comment:
nit: does `writeToAudioStreamAndClose()` take a noticeable amount of time? I'm wondering if we should wait to send the image until we have the audio ready (this may become irrelevant when we change how the data is stored).

Reply:
Good question - I'll move the `sendMessage` calls together for now at least.